health_checks.go

```diff
@@ -0,0 +1,201 @@
+package rabbithole
+
+import "strconv"
+
+type TimeUnit string
+
+const (
+	SECONDS TimeUnit = "seconds"
+	DAYS    TimeUnit = "days"
+	MONTHS  TimeUnit = "months"
+	YEARS   TimeUnit = "years"
+)
+
+type Protocol string
+
+const (
+	AMQP091   Protocol = "amqp091"
+	AMQP10    Protocol = "amqp10"
+	MQTT      Protocol = "mqtt"
+	STOMP     Protocol = "stomp"
+	WEB_MQTT  Protocol = "web-mqtt"
+	WEB_STOMP Protocol = "web-stomp"
+)
+
+// Health represents response from healthchecks endpoint
+type Health struct {
+	Status    string   `json:"status"`
+	Reason    string   `json:"reason"`
+	Missing   string   `json:"missing"`
+	Ports     []string `json:"ports"`
+	Protocols []string `json:"protocols"`
+}
+
+// Responds a 200 OK if there are no alarms in effect in the cluster, otherwise responds with a 503 Service Unavailable.
```

Comment:
"Responds with". I think a health check response interface would be worth adding so that the users don't have to manually check the code.

Reply:
Ah, yes. I blindly took the documentation from the api documentation:

> /api/health/checks/alarms | Responds a 200 OK if there are no alarms in effect in the cluster, otherwise responds with a > 503 Service Unavailable.
> -- | --